indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -151,6 +152,7 @@ export default function FormItem({
 
   return (
     <div
+      data-html-name={htmlName}
```

Comment:
Just wondering... is it useful to always have it, since for non-personal-data-fields it's something like `field_12345`. OTOH, maybe someone wants to use custom CSS to target it anyway.